outbound_sms.py

```diff
@@ -1,36 +1,37 @@
 from serverless_sdk import tag_event
 from clients import twilio, kinesis
+from utils.sqs import get_payloads_from_sqs_event
 
 
 def send_message(event, context):
     tag_event("send_message", "raw_event", event)
 
-    response = twilio.send_message(event["To"], event["Body"])
-
-    tag_event(
-        "send_message",
-        "twilio_response",
-        {
-            "twilio_message_id": response.sid,
-            "to": response.to,
-            "body": response.body,
-            "status": response.status,
-            "error_code": response.error_code,
-            "error_message": response.error_message,
-        },
-    )
-
-    kinesis.publish_outbound_message(
-        {
-            "type": "OUTBOUND_SMS",
-            "payload": {
-                "MessageSid": response.sid,
-                "To": response.to,
-                "Body": response.body,
-                "MessageStatus": response.status,
+    messages = get_payloads_from_sqs_event(event)
```

Comment:
We will need to be able to specify delays on a per-SMS basis. So I’d expect to pull that off of the SQS message.

(E.g. see the simulator code. For some dialog event types we will have different delays)

Reply:
My thinking is we would add a `DelaySeconds` argument when we enqueue to SQS rather than doing that in the handler. We would add it here: https://github.com/jeffreysilver/stopcovid-lambdas/pull/10/files#diff-19ec384d11cc67cd9535d36f93168d20R13

That way the send message lambda doesnt have to know about delays

Reply:
yes, misunderstood your code. Sorry. I agree that we would add the delay on the enqueueing side and not on the dequeuing side.

Do we agree that we'd eventually enqueue the SMS commands not from the command stream handler but from the dialog events handler?

Reply:
Yes! We definitely shouldnt be enqueuing here in the long run. Just doing this for now since its helpful to test the entire circuit